src/graphql/mod.rs

```diff
@@ -3,13 +3,13 @@ pub mod relay;
 use async_graphql::{EmptySubscription, MergedObject};
 
 use crate::modules::auth::graphql::AuthMutation;
-use crate::modules::post::graphql::{PostMutation, PostQuery};
+// use crate::modules::post::graphql::{PostMutation, PostQuery};
 use crate::modules::user::graphql::{UserMutation, UserQuery};
 
 #[derive(MergedObject, Default)]
-pub struct Query(pub PostQuery, pub UserQuery);
+pub struct Query(/*pub PostQuery*/ pub UserQuery);
 
 #[derive(MergedObject, Default)]
-pub struct Mutation(pub AuthMutation, pub PostMutation, pub UserMutation);
+pub struct Mutation(pub AuthMutation, /*pub PostMutation*/ pub UserMutation);
```

Comment:
@Dave136 as Posts are not yet being used I felt free to comment them in order to test changes.
As you work on #7 , feel free to uncomment this and add `scope` similar to `Gender` and `Pronoun`.